src/test/http-mocks/idam.ts

```diff
@@ -23,6 +23,12 @@ export function resolveExchangeCode (token: string) {
     })
 }
 
+export function resolveInvalidateCode (token: string) {
```

Comment:
When main method is renamed please do follow-up rename here as well.

Reply:
done